src/core/Value.cc

```diff
@@ -635,6 +635,61 @@ void VectorType::VectorObjectDeleter::operator()(VectorObject *v)
   delete orig;
 }
 
+std::shared_ptr<Values>
+VectorType::embeddedValues() const
+{
+  class VectorValueIterator: public VirtualValueIterator  {
```

Comment:
I think these local classes causes visibility issues, and leads to linker issue in the CI.